app/src/main/java/com/dodo/flashcards/util/Screen.kt

```diff
@@ -25,5 +25,5 @@ sealed class Screen(val route: String) {
         }
     }
 
-    override fun toString():String = route
+    override fun toString() : String = route
```

Comment:
Nit, change to `override run toString(): String = route`, no space needed before